src/openzaak/components/documenten/query.py

```diff
@@ -76,7 +76,7 @@ def filter_for_authorizations(
             )
             vertrouwelijkheidaanduiding_whens.append(
                 When(
-                    **{"informatieobjecttype": informatieobjecttype},
+                    **{"_informatieobjecttype": informatieobjecttype},
```

Comment:
@joeribekker @sergei-maertens 
This change is just workaround. After adding all loose-fk fields the authorization should be modified.
But this can be tricky since in the `AuthorizationsFilterMixin` we directly construct DB queries

Reply:
@annashamray can you create a follow up issue to validate permissions with remote types?

Reply:
Issue - https://github.com/open-zaak/open-zaak/issues/284